### PR TITLE
Unsplash last 3 images history

### PR DESCRIPTION
**Pull request by Kalovelo**
Refs #134.
Image blobs are being cached on fetch and shown under unsplash settings. Current limit is 3, and working on Unsplash. 
I could work on giphy too.

🎉 Hello Tabliss team, this is my first contribution here! I have to say I love this plugin, can't wait to roll on the new features. 🎉

### CODE REVIEW

File: src/plugins/backgrounds/unsplash/Unsplash.tsx [hunk 1]
```diff
@@ -14,23 +13,33 @@ const Unsplash: FC<Props> = ({
...
+        };
+      }),
     cacheArea,
     data.timeout * 1000,
     [data.by, data.collections, data.featured, data.search],
   );
 
-  const url = useObjectUrl(image && image.data);
-
+  const url = cache && URL.createObjectURL(cache.now.currentImage.data);
```
Comment (trickypr) [inline src/plugins/backgrounds/unsplash/Unsplash.tsx]: I ran into a few errors with this line. If you run into a type error with this line, you will have to clear site data or open it in an unused profile. Maybe there should be some form of fallback

**Comment by trickypr** [comment]
On edge (chromium) about every second the background goes black for a frame. I think this has something to do with the background image's source changing every second and edge (chromium) not being able to fetch it fast enough.

Some frames with black images

**Comment by Kalovelo** [comment]
Thank you @trickypr for your reply.

Regarding the error with URL, i fixed it with your suggestion, using a fallback.

I will check the rest of browsers and commit back as soon as possible.

**Comment by Kalovelo** [comment]
I fixed the compatibility errors,
new PR : https://github.com/joelshepherd/tabliss/pull/191

**Comment by gonzalrick** [comment]
Hi @Kalovelo , thanks for adding this feature. What is the status of this PR?

**Pull request closed without merging**